### PR TITLE
test: integration test for Docker proxy push feature

### LINKED ISSUE #511: DockerProxyIT.java:51-53: Add integration test for Docker...
The puzzle `499-4f132183` from #499 has to be resolved: 

https://github.com/artipie/artipie/blob/ab444fd0b00d0dbe4240648afb1f6fdd8ad95fc2/src/test/java/com/artipie/docker/DockerProxyIT.java#L51-L53

The puzzle was created by Oleg Mozzhechkov on 25-Aug-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by dgarus**
Closes #511

### CODE REVIEW

**Comment by dgarus** [comment]
@artipie/maintainers Hello!
Could anybody do the review, please?

File: src/test/java/com/artipie/docker/DockerProxyIT.java [hunk 1]
```diff
@@ -100,4 +97,60 @@ void shouldPullRemote() throws Exception {
...
+        this.deployment.assertExec(
+            "Failed to push image to Artipie",
+            new ContainerResultMatcher(
+                new IsEqual<>(ContainerResultMatcher.SUCCESS),
+                new StringContains(
+                    "The push refers to repository [artipie:8080/my-docker/alpine]"
+                )
+            ),
+            "docker", "push", image
+        );
```
Comment (g4s8) [inline src/test/java/com/artipie/docker/DockerProxyIT.java:136]: Why does it work? You are pushing image to proxy registry with `registry-1.docker.io` and `mcr.microsoft.com` remotes, but the proxy doesn't have push permissions in these remotes, isn't it?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@dgarus one questions

File: src/test/java/com/artipie/docker/DockerProxyIT.java (hunk 1 above)
Comment (dgarus) [reply src/test/java/com/artipie/docker/DockerProxyIT.java:136]: Sure it doesn't have; it pushes an image to the local storage as declared in the [doc](https://github.com/artipie/artipie/tree/master/examples/docker#docker-proxy-repo).
The task description is 
`Docker proxy supports pushing to local storage if such storage is specified. 
 It should be verified that an image can be pushed to a proxy repository and pulled later`.
It looks like we should test `pushing to local storage,` so I think all is ok.

**Review by g4s8 — APPROVED** [review]
@dgarus thanks

**Pull request merged**